ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -288,6 +289,26 @@ void OwInterface::managePanTilt (const string& opname,
 
   int id = Running.at (opname);
 
+  //if position is over 360 we want to bring it back within the
+  //-360 to 360 range to check if goal position has been reached.
+  if(fabs(current) > 360){
+    if(current < 0){
+      current = fmod(fabs(current),360.0)*-1;
+    }
+    else{
+      current = fmod(fabs(current), 360.0);
+    }
+  }
+
+  //We cant guarantee which way the antenna will move, so we have to 
+  //check if the current angle is equivalent.Ex:-180 degrees == 180 degrees.
+  if(current > 0 && goal < 0){
+    current = current - 360; 
+  }
+  else if(current < 0 && goal > 0){
+    current = current + 360;
+  }
+
```

Comment:
I'm concerned that the local variable 'current' is manipulated/normalized here, but that its source, member variable m_current, is unaffected.   Perhaps it doesn't matter.   But there are two approaches that would remove this confusion:

1. Remove the 'current' and 'goal' parameters, and have m_current and m_goal be used directly. 
2. Have 'current' be passed by reference.

I think (1) is much better.   If m_current should not be updated by this function, then don't change anything, but I'd be curious to know why.

Reply:
We could definitely do that. If we do normalize m_current I think it could be useful to normalize the m_goal parameter as well so that no matter what a user enters it always will work. The only downside is that the actual unnormalized joint angle is inaccessible from PLEXIL, but if that ever became a problem it would be a pretty easy fix.

Reply:
I don't know anything about real-world use cases here.  I'll suggest leaving this be, and creating a low-priority Jira to look at this issue.  It could be called "Investigate normalization of command/actual antenna angles".

Reply:
Sounds good!